src/errors/InternalError.ts

```diff
@@ -1,4 +1,4 @@
-import { isError } from '../utils/typeUtils'
+import { isNativeError } from 'node:util/types'
```

Comment:
why this change? this method was deprecated AFAIR

Reply:
It's not depreciated: https://nodejs.org/api/util.html#utiltypesisnativeerrorvalue
I think that it's better to use std lib solutions

Reply:
ok, then it makes sense